config/public-dataset.yaml

```diff
@@ -104,6 +103,15 @@ step1:
   gatk_vcf_indir: '{general.data_root}/test_data_from_1000_genomes/'
   gatk_mt_outfile: '{mtdir}/gatk_unprocessed.mt'
 
+  # --- Step 1.2 --- #
+  validate_verifybamid:
+    verifybamid_selfsm: '{metadir}/test_data_from_1000_genomes.verify_bam_id_result.selfSM.tsv'
```

Comment:
perhaps add comments about setting it to null if not run?

Reply:
Yes, a great idea. Added.